### DISCUSSION TITLE
文章中代码块缩进问题

### DISCUSSION THREAD

**Discussion by ctdrm** [MaLuns/hexo-theme-async#53] [Q&A 疑难杂症] [answered]
![image](https://user-images.githubusercontent.com/100547590/212289882-aedd632a-b0fe-4a83-9889-02c249a7087f.png)
![image](https://user-images.githubusercontent.com/100547590/212290019-43fa4705-6941-4b73-a4ca-613973f65883.png)

佬佬，本地的md文件中插入代码后，一个缩进是4个空格，在博客界面中会显示7个空格，该如何调整？

**Comment by MaLuns** [comment, marked as answer]
heox 应该不会改动这个，应该和你编辑器有关系，是不是你缩进使用了 tab 啥的？

**Reply by ctdrm** [reply]
懂了，我使用空格代替tab后正常了，谢谢大佬

**Reply by MaLuns** [reply]
> 懂了，我使用空格代替tab后正常了，谢谢大佬

如果你习惯是使用tab，可以用 css 的 tab-size 控制页面里的缩进，浏览器了默认一个 tab 是 8 空格

**Comment by JingxuGu** [comment]
> 可以用 css 的 tab-size 控制页面里的缩进

这个在哪里呢？没有找到
